sdk/python/packages/flet-cli/src/flet_cli/commands/build.py

```diff
@@ -32,7 +32,7 @@
 
 PYODIDE_ROOT_URL = "https://cdn.jsdelivr.net/pyodide/v0.25.0/full"
 DEFAULT_TEMPLATE_URL = "gh:flet-dev/flet-build-template"
-MINIMAL_FLUTTER_VERSION = "3.24.0"
+MINIMAL_FLUTTER_VERSION = "3.27.0"
```

Comment:
Do you think we should check exact version like `3.27.{anything}` rather than "greater or equal"?

Reply:
Yeah, we can do that.

Reply:
Let's do it in this PR then?